dags/top-public-synapse-projects-30-days-from-snowflake.py

```diff
@@ -31,6 +31,7 @@
 }
 
 SYNAPSE_RESULTS_TABLE = "syn55382267"
+SYNAPSE_HOMEPAGE_PROJECT_ID = 23593546
```

Comment:
Would it make sense to make this an array, in the event we need to add other projects it would be a simple add?

Reply:
I think that would be easy enough to do in the future if we needed to, but for now I like the specificity of the named constant so it's clear what is being filtered out.

Reply:
Works for me!